modules/client/test/integration/client/estimation.test.ts

```diff
@@ -37,7 +40,17 @@ describe("Client", () => {
       const daoCreationParams: ICreateParams = {
         metadataUri: `ipfs://QmeJ4kRW21RRgjywi9ydvY44kfx71x2WbRq7ik5xh5zBZK`,
         ensSubdomain: daoName.toLowerCase().replace(" ", "-"),
-        plugins: [{ id: "0x1234", data: new Uint8Array() }],
+        plugins: [
+          {
+            id: deployment.addressListRepo.address,
+            data: toUtf8Bytes(
+              defaultAbiCoder.encode(
+                ["uint64", "uint64", "uint64", "address[]"],
+                [1, 1, 1, []],
```

Comment:
I guess these `1, 1, 1` don't alter the estimation, but don't we have the real values?

Reply:
Sure, I can change it to the real values